### PR TITLE
Add new test environment TESTER-MOCK + clean up (DEV, EXPOSUREAPP-6138)

### PR DESCRIPTION
**Pull request by d4rken**
* Additional environment option that was requested
* Remove previously added presence tracing entries that are no longer needed
* Show public keys in test fragment
* Compare with https://github.com/corona-warn-app/cwa-app-android-environments/pull/1

### CODE REVIEW

**Review by harambasicluka — COMMENTED** [review]
Tested, coding looks good. But we need to verify if the downloading still works as @LukasLechnerDev has implemented it.

File: Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/EnvironmentState.kt [hunk 1]
```diff
@@ -9,8 +9,9 @@ data class EnvironmentState(
     val urlDownload: String,
     val urlVerification: String,
     val urlDataDonation: String,
-    val urlQrCodePosterTemplate: String
-
+    val logUpload: String,
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/EnvironmentState.kt:12]: `logUpload` => `urlLogUpload` to be consistent with the other namings?

**Review by LukasLechnerDev — DISMISSED** [review]
LGTM; Downloading the Template with MockServer still works

**Review by harambasicluka — DISMISSED** [review]

File: Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/EnvironmentState.kt (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/EnvironmentState.kt]: Good idea. Fixed it.

**Review by LukasLechnerDev — APPROVED** [review]

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**